### DISCUSSION TITLE
blog/posts/230206-gatsby-7-sticky-toc/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#115] [Giscus]
# blog/posts/230206-gatsby-7-sticky-toc/

이전 글들에서는 Gatsby 기본 기능을 구현을 위한 튜토리얼을 소개하며 태그 기능을 구현했고, MDX 글 안에 인라인 이미지와 테이블을 삽입했으며, 시리즈 기능도 만들어보았고, 임시글 기능도 구현했고, Google Analytics 도 적용해보았으며…

https://ricale.kr/blog/posts/230206-gatsby-7-sticky-toc/

**Comment by Byongho96** [comment]
오 toc 글 보려고 들어왔다가, 바로 위에 좋아요(반응) 기능 보고 놀랐어요!!
혹시 어떻게 구현하셨을까요?? 따로 서버를 돌리고 있는건가요?

**Reply by ricale** [reply]
[giscus](https://giscus.app/ko)를 사용했습니다 (GitHub 기반이라 서버는 따로 필요없습니다). 공식 문서가 잘 정리되어 있어서 따라하시면 쉽게 하실 수 있을 거에요. ([이 글](https://ricale.kr/blog/posts/230211-gatsby-8-comments/)에 어떻게 적용했는지도 정리해 두었습니다)

**Reply by Byongho96** [reply]
오오오 감사합니다~~
